packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/analytics.ts

```diff
@@ -1,47 +1,54 @@
 import { queuevent } from '@blockchain-com/constellation'
-import type { AnalyticsPayload } from 'middleware/analyticsMiddleware/types'
+import type { AnalyticsValue } from 'middleware/analyticsMiddleware/types'
 import { AnalyticsKey } from 'middleware/analyticsMiddleware/types'
 import { generateUniqueUserId } from 'middleware/analyticsMiddleware/utils'
-import { v4 } from 'uuid'
 
-const queueCallback = async (rawEvents) => {
+const queueCallback = async (
+  rawEvents: {
+    key: AnalyticsKey
+    payload: AnalyticsValue
+  }[]
```

Comment:
maybe to export this to be type and the use it here like
```
type RawEventsType = {
    key: AnalyticsKey
    payload: AnalyticsValue
  }
```

Reply:
Good one!